docs/src/helpers/logo-helper.js

```diff
@@ -17,10 +17,18 @@
  */
 
 import BackpackLogoWhite from '../static/backpack-logo-white.svg';
+import BackpackLogoGrey from '../static/backpack-logo-sky-grey.svg';
```

Comment:
while we're at this, can we rename these to `backpack-logo-light` and `backpack-logo-dark`?

Reply:
So these are in line with the spec from Will so will need to check as its not really based on light and dark settings more what page is shown on